src/store.ts

```diff
@@ -11,35 +11,62 @@ interface ICycle {
 }
 
 const startingCycles: ICycle[] = [
-  {cursor: 2, length: 2},
+  { cursor: 1, length: 2 },
+  { cursor: 3, length: 3 },
 ];
 
 export default new Vuex.Store({
   state: {
     asm: {
       activators: 1,
-      // cycles: new Array<ICycle>(),
+      nonActivators: 1,
       cycles: startingCycles,
       iteration: 0,
     },
   },
   getters: {
     anyActive(state): boolean {
-      return _.some(state.asm.cycles, (cycle) => {
```

Comment:
@navyhockey56 Not a fan of lodash?  It has great array methods, make js feel like ruby.

Reply:
@sballan For whatever reason, when using _.some, it doesn't work correctly (even though my interpertation of what .some should do is exactly how I wrote the loop)

Reply:
@navyhockey56 Hmm - did you `return` in the callback?  After writing ruby that's something I tend to forget:
```
return _.some(state.asm.cycles, (cycle) => {
  return ...
})
```